docs/upgrade.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade
```

Comment:
Plz modify the `manifest.json` to include this new page. 

You can make sure your docs contribution work with [this simple coder.com workflow](https://github.com/coder/coder.com#docs-development).

Reply:
I always forget to update it 😅

Reply:
or are you referring to nesting it under the `/install` directory?

Reply:
I meant that this should be a new doc page under the Installation category.